<?xml version="1.0" encoding="utf-8"?>
<launch>

    <!--<param name ="/use_sim_time" value="true"/>-->
    <param name="left_img_topic" type="string" value="/cam00/image_raw"/>
    <param name="right_img_topic" type="string" value="/cam01/image_raw"/>
    <param name="calib_width" type="int" value="1392"/>
    <param name="calib_height" type="int" value="512"/>

    <param name="calib_file_name" type="string" value="/home/usl/catkin_ws/src/stereo_dense_reconstruction/calibration/kitti_0027.yml"/>
    <node  pkg="stereo_dense_reconstruction"
           name="stereo_dense_reconstruction"
           type="dense_reconstruction"
           output="screen">
    </node>

</launch>
